apps/files_sharing/src/views/SharingInherited.vue

```diff
@@ -37,11 +36,19 @@
 		</SharingEntrySimple>
 
 		<!-- Inherited shares list -->
-		<SharingEntryInherited v-for="share in shares"
+		<div v-if="loaded && shares.length === 0" class="sharing-entry__desc">
+			<span class="sharing-entry__title">{{ 'No others with access' }}</span>
```

Comment:
Sorry, missed this label. This need the same pattern as the other one:

```suggestion
			<span class="sharing-entry__title">{{ t('files_sharing', 'No others with access') }}</span>
```

Reply:
@artonge  Should be good now!